src/test/java/seedu/storemando/logic/commands/CommandTestUtil.java

```diff
@@ -26,18 +26,18 @@
  */
 public class CommandTestUtil {
 
-    public static final String VALID_NAME_AMY = "Amy Bee";
-    public static final String VALID_NAME_BOB = "Bob Choo";
-    public static final String VALID_QUANTITY_AMY = "11111111";
-    public static final String VALID_QUANTITY_BOB = "22222222";
-    public static final String VALID_EXPIRYDATE_AMY = "amy@example.com";
-    public static final String VALID_EXPIRYDATE_BOB = "bob@example.com";
-    public static final String VALID_LOCATION_AMY = "Block 312, Amy Street 1";
-    public static final String VALID_LOCATION_BOB = "Block 123, Bobby Street 3";
-    public static final String VALID_TAG_HUSBAND = "husband";
-    public static final String VALID_TAG_FRIEND = "friend";
-
-    public static final String NAME_DESC_AMY = " " + PREFIX_NAME + VALID_NAME_AMY;
+    public static final String VALID_NAME_STRAWBERRY_MILK = "Strawberry Milk";
```

Comment:
Some variable changes are still seen here and there

Reply:
I also think it should be changed back to VALID_NAME_AMY for now just for consistency.

Reply:
Fixed.